### PR TITLE
added lcm

### PR DESCRIPTION
**Pull request by awesomekoder95**

### CODE REVIEW

File: pydsa/lcm.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+"""
+Lowest Common Multiple
+----------------------
+
+A very simple implementation of LCM 
+
+Pseudo Code: https://en.wikipedia.org/wiki/Least_common_multiple
+"""
+
+def lcm(a,b):
```
Comment (CuriousLearner) [inline pydsa/lcm.py]: Does not follow PEP8

4 spaces indentation

File: pydsa/tests/test_lcm.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+from pydsa.lcm import lcm
+
+def test_gcd():
+    assert lcm(4, 10) == 20
+    assert lcm(80, 25) == 400
+    assert lcm(41, 12) == 492
+    assert lcm(10251, 210) == 717570
+    assert lcm(126, 105) == 630
+    assert lcm(145,780) == 22620
```
Comment (CuriousLearner) [inline pydsa/tests/test_lcm.py]: space after comma  `,`

File: pydsa/tests/test_lcm.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+	
+	
+
+
```
Comment (CuriousLearner) [inline pydsa/tests/test_lcm.py]: Also no need to leave so many blank lines

**Comment by awesomekoder95** [comment]
please review @CuriousLearner  , changes added to code

File: pydsa/__init__.py [hunk 4]
```diff
@@ -17,3 +17,4 @@
 from .gnome_sort import gnome_sort
 from .mod_inverse import mod_inverse
 from .euclid import gcd, extended_euclid
+from .lcm import lcm
```
Comment (CuriousLearner) [inline pydsa/__init__.py]: Leave a line after end of the file.

**Pull request closed without merging**